.github/workflows/build-linux.yml

```diff
@@ -19,38 +19,112 @@ on:
         type: boolean
```

Comment:
Please ignore any changes in this file, it is unused in the workflow. This contains some very early experiments.